src/functions/DidSiopMetadata.ts

```diff
@@ -31,16 +162,18 @@ function verifySubjectIdentifiers(subjectIdentifiers: SubjectIdentifierType | Su
   return subjectIdentifiers === SIOP.SubjectIdentifierType.DID;
 }
 
-function verifyDidMethods(methodsA: string[] | string, methodsB: string[] | string): void {
+function verifyDidMethods(methodsA: string[] | string, methodsB: string[] | string): Array<string> {
```

Comment:
suggest to rename to supportedDidMethods

Reply:
Fixed!